locales/quest.pot

```diff
@@ -0,0 +1,92 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2017 Puppet, LLC.
+# This file is distributed under the same license as the Quest package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Quest \n"
+"Report-Msgid-Bugs-To: learningvm@puppet.com\n"
+"POT-Creation-Date: 2017-06-14 10:57-0700\n"
+"PO-Revision-Date: 2017-06-14 10:57-0700\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```

Comment:
@greglarkin For the moment, the scope is limited to a working implementation of the gettext-setup gem. When we set up the translation project in Transifex, I'll work w/ Larissa to ensure these details are correct.